src/morale.cpp

```diff
@@ -480,7 +480,8 @@ void player_morale::display( double focus_gain )
             const int decimals = ( value - static_cast<int>( value ) != 0.0 ) ? 2 : 0;
             color = ( value > 0.0 ) ? c_green : c_red;
             mvwprintz( w, y, getmaxx( w ) - 8, color, "%+6.*f", decimals, value );
-        } else {
+        } else
+        {
```

Comment:
Tried moving the curly to its proper place after the `if` a few lines above - onto the second line. `make astyle` moves it right back where it is now. :/

I now think I should close this PR, until someone merges it by accident.